Samples/Excel.OfflineStorageAddin/README.md

```diff
@@ -0,0 +1,126 @@
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Excel.OfflineStorageAddin | Nancy Wang, Albert Dotson (**Microsoft**)
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0  | (update later) | Initial release
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+----------
+# Scenario: Storing data using local storage
+This sample Office add-in inserts a table of fictitious basketball players' stats in your file, retrieved from a local file named `sampleData.json`. In this sample code, data from the add-in is stored in `localStorage` to allow users who previously opened the add-in with online connection to insert the table of stats offline.
+
+While this add-in gets its data from a local server, implementation of `localStorage` as shown in this sample can be extended to add-ins that get their data from online sources. Furthermore, although this sample runs only in Excel, `localStorage` can be used to offline data across Word, Excel, and PowerPoint.
+
+**Note**: `localStorage` can store up to 5MB of data. To store larger amounts of data offline and for improved performance, consider using [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API). Note that as of now, IndexedDB isn't supported by all browsers used by Office add-ins and may cause your add-in to fail on some computers. 
+Furthermore, `localStorage` is not secure and should only be used for data that can be made publicly available. To protect add-in data while offline, consider using offline cookies.
+As another option, you can store your add-in's data in [`Office.Settings`](https://docs.microsoft.com/en-us/javascript/api/office/office.settings?view=office-js). Using Office.Settings will enable your add-in's offline capabilities to persist within the file (e.g. if you'd like to share your file with others).
+
+## Build and run the sample
+
+1. Clone or download this repository. 
+2. In the command line, navigate to the **Excel.OfflineStorageAddin** folder from your root directory.
+
+The following code sample shows you how to do these two steps: 
+```command&nbsp;line
+
+$ git clone https://github.com/OfficeDev/PnP-OfficeAddins.git
+$ cd PnP-OfficeAddins
+$ cd Samples
+$ cd Excel.OfflineStorageAddin
+```
+You can try out this sample by running the following commands:
+```command&nbsp;line
+# this will download the node modules needed to run this add-in
+$ npm install
+
+# this will build the add-in 
+$ npm run build
+
+# this will start the server to run your add-in on Excel on your desktop

```

Comment:
This comment isn't quite accurate. "npm start" will start the dev-server and also launch Excel by opening an Excel document that loads your add-in.

Reply:
Good point, I'll update this to more accurately reflect what's going on. Thanks Adam!